e2e-test/src/test/java/io/cdap/plugin/normalize/actions/NormalizeActions.java

```diff
@@ -0,0 +1,116 @@
+
+  static {
+    SeleniumHelper.getPropertiesLocators(NormalizeLocators.class);
+  }
+
+  public static void selectOutputSchemaAction(String action) {
+    ElementHelper.selectDropdownOption(CdfSchemaLocators.schemaActions,
+            CdfPluginPropertiesLocators.locateDropdownListItem(action));
+    WaitHelper.waitForElementToBeHidden(CdfSchemaLocators.schemaActionType(action));
+  }
+
+  public static void clickOutputSchemaDeleteRowButton(int row) {
+    ElementHelper.clickOnElement(NormalizeLocators.outputSchemaDeleteRowButton(row - 1));
+  }
+
+  public static void enterFieldsMapping(String jsonFieldsMapping) {
+    Map<String, String> fieldsMapping =
+      JsonUtils.convertKeyValueJsonArrayToMap(PluginPropertyUtils.pluginProp(jsonFieldsMapping));
+    int index = 0;
+    for (Map.Entry<String, String> entry : fieldsMapping.entrySet()) {
+      ElementHelper.sendKeys(NormalizeLocators.fieldsMappingKey(index), entry.getKey());
+      ElementHelper.sendKeys(NormalizeLocators.fieldsMappingValue(index), entry.getValue());
+      ElementHelper.clickOnElement(NormalizeLocators.fieldsMappingAddRowButton(index));
+      index++;
+    }
+  }
+
+  public static void enterOutputSchema(String jsonOutputSchema) {
+    Map<String, String> outputSchema =
+      JsonUtils.convertKeyValueJsonArrayToMap(PluginPropertyUtils.pluginProp(jsonOutputSchema));
+    int index = 0;
+    for (Map.Entry<String, String> entry : outputSchema.entrySet()) {
+      ElementHelper.sendKeys(NormalizeLocators.outputSchemaFieldName(index), entry.getKey());
+      ElementHelper.clickOnElement(NormalizeLocators.outputSchemaDataTypeDropdown(index));
+      SeleniumDriver.getDriver()
+        .findElement(NormalizeLocators.outputSchemaDataTypeOption(index, entry.getValue())).click();
+      ElementHelper.clickOnElement(NormalizeLocators.outputSchemaAddRowButton(index));
+      index++;
+    }
+  }
+
+  public static void enterFieldsToBeNormalized(String jsonFieldsToBeNormalized) {
+    Map<String, String> fieldsToBeNormalized =
+      JsonUtils.convertKeyValueJsonArrayToMap(PluginPropertyUtils.pluginProp(jsonFieldsToBeNormalized));
+    int index = 0;
+    for (Map.Entry<String, String> entry : fieldsToBeNormalized.entrySet()) {
+      ElementHelper.sendKeys(NormalizeLocators.fieldNormalizingNormalizeField(index), entry.getKey().split("#")[0]);
+      ElementHelper.sendKeys(NormalizeLocators.fieldNormalizingFieldName(index), entry.getKey().split("#")[1]);
+      ElementHelper.sendKeys(NormalizeLocators.fieldNormalizingFieldValue(index), entry.getValue());
+      ElementHelper.clickOnElement(NormalizeLocators.fieldsNormalizeAddRowButton(index));
+      index++;
+    }
+  }
+
+  public static int getTargetRecordsCount() {
+    String incount = NormalizeLocators.targetRecordsCount.getText();
+    return Integer.parseInt(incount.replaceAll(",", ""));
+  }
+
+  public static void verifyNormalizePluginPropertyInvalidInlineErrorMessageForRow(String propertyName, int row,
```

Comment:
Remove `Invalid` word from name

Check for locator and color method name as well

Reply:
Removed